go/vt/vttablet/tabletmanager/action_agent.go

```diff
@@ -109,6 +109,12 @@ type ActionAgent struct {
 	// only used if exportStats is true.
 	statsTabletTypeCount *stats.CountersWithSingleLabel
 
+	// statsBackupIsRunning is set to true if an online backup is running
+	// not set if an offline backup is running because that is
+	// indicated by tablet_type = BACKUP
+	// only used if exportStats is true
+	statsBackupIsRunning *stats.String
```

Comment:
Usually a boolean metric would be expressed as a Gauge that is either 0 or 1. That makes it easy to construct monitoring queries that use these metric values as part of compound boolean expressions.

I think it would also be nice to include both online and offline backups in this metric. We can use [GaugesWithSingleLabel](https://godoc.org/vitess.io/vitess/go/stats#GaugesWithSingleLabel) and have a label named `mode` whose value is either `online` or `offline` depending on the result of `ShouldDrainForBackup`.

Then in monitoring we can aggregate or slice-and-dice on this metric however we want.